src/revolve/api.py

```
@@ -0,0 +1,290 @@
+
+    def get_file(self, req, resp):
+        file_name = req.get_param("name")
+        content = read_python_code(file_name)
+        if file_name.endswith(".py"):
+            content = f"```python\n{content}\n```"
+        elif file_name.endswith(".json"):
+            content = f"```json\n{content}\n```"
+        resp.status = falcon.HTTP_200
+        resp.media = {"content": content}
+
+    def get_file(self, req, resp):
```

Ruff (0.11.9)

Redefinition of unused `get_file` from line 156

(F811)